doc/source/whatsnew/v1.4.0.rst

```diff
@@ -540,6 +540,7 @@ Other Deprecations
 - Deprecated parameter ``names`` in :meth:`Index.copy` (:issue:`44916`)
 - A deprecation warning is now shown for :meth:`DataFrame.to_latex` indicating the arguments signature may change and emulate more the arguments to :meth:`.Styler.to_latex` in future versions (:issue:`44411`)
 - Deprecated :meth:`Categorical.replace`, use :meth:`Series.replace` instead (:issue:`44929`)
+- Deprecated :meth:`Index.__getitem__` with a bool key; use ``index.values[key]`` to get the old behavior (:issue:`44051`)
```

Comment:
do we have other deprecated indexing like this? can you co-locate (followup ok too)

Reply:
i think from a previous version, so the note will be in a different file

Reply:
kk that's fine then